src/views/partials/error-summary.njk

```diff
@@ -24,6 +24,6 @@
   </div>
 </div>
 <script>
-  document.title = "Error - " + document.title;
+  document.title = "{{ i18n.individual_statement_error_title }}" + document.title;
```

Comment:
won't this be used on other screens too?  ought it to be in errors json, same as the other i18n values already in this template?

Reply:
Changes made, cheers.